metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -273,19 +274,32 @@ def _get_columns_to_profile(self) -> List[str]:
 
         # Compute columns to profile
         columns_to_profile: List[str] = []
+
         # Compute ignored columns
-        ignored_columns: List[str] = []
-        for col in self.dataset.get_table_columns():
+        ignored_columns_by_pattern: List[str] = []
+        ignored_columns_by_type: List[str] = []
+
+        types_to_ignore = _get_column_types_to_ignore(self.dataset.engine.dialect.name)
+
+        for col_dict in self.dataset.columns:
```

Comment:
were we previously calling `get_table_columns` twice? and now we're using a cached result instead?

Reply:
Earlier, get_table_columns was returning from self.columns (cached result)only,  but it returned only column names (List[str]).